content/post/81-js-libp2p-0.25.md

```diff
@@ -0,0 +1,119 @@
+---
+date: 2019-04-12
+url: 81-js-libp2p-0-25
+title: js-libp2p 0.25.0 released
+author: Jacob Heun
+---
+
+> Save the routers, save the world.
+
+# 🔦 Highlights
+
+Libp2p 0.25 is already in the new [js-ipfs 0.35 release](https://github.com/ipfs/js-ipfs/issues/1826). If you're using js-ipfs, you'll get these changes immediately if you upgrade there!
+
+## ☎️ Auto Dial
+Libp2p will now auto dial to discovered peers by default. This can be disabled in configuration if you need to do any specialized dialing. Discovered peers will only be dialed if we are under our min peers threshold, which is configured for the Connection Manager. You can read more about [Peer Discovery and Auto Dial here](https://github.com/libp2p/js-libp2p/blob/v0.25.0/PEER_DISCOVERY.md).
+
+## ✂️ Better Connection Management
+The Libp2p Switch was doing a bad job of letting the Connection Manager know just how many connections there were. This resulted in the Connection Manager thinking there were far fewer connections than there actually were. This has been fixed, and now the Connection Manager is back to keeping your router safe from the connection plague.
```

Comment:
I'd rephrase this around the positive, i.e. remove "The Libp2p Switch was doing a bad job" and focus on the fact that there were bugs that were fixed.